Improve interview documentation (#208)
AY2223S1-CS2103-F14-3/tp: docs/UserGuide.md

### Adding / Editing an interview: `interview`

<div markdown="span" class="alert alert-primary">:bulb: <b>Tip:</b><br>

> * Interview, being one of the branches of the application process, plays a crucial part in every application. It helps user to have a better clarity on understanding how's the current application process going on.

Reviewer: ```suggestion
* Interview, being one of the branches of the application process, plays a crucial part in every application. It helps users have a better understanding on how the current application process is going on.
```
